tests/src/unit-regression2.cpp

```diff
@@ -36,7 +36,7 @@ using ordered_json = nlohmann::ordered_json;
     #include <variant>
 #endif
 
-#ifdef JSON_HAS_CPP_20
+#ifdef JSON_HAS_CPP_20 && __has_include(<span>)
```

Comment:
The proper form for this is `#if defined(JSON_HAS_CPP_20) && ...

Reply:
Maybe I should sleep instead of writing code while tired...